lib/menu.ts

```diff
@@ -190,18 +214,22 @@ function findAncestorChild(ancestor: Element, elem: Element|null): Element|null
 /**
  * Implements a menu item which opens a submenu.
  */
-export function menuItemSubmenu(submenu: MenuCreateFunc, ...args: DomElementArg[]): Element {
+export function menuItemSubmenu(
+  submenu: MenuCreateFunc,
+  options: ISubMenuOptions,
```

Comment:
Oh, this is kind of annoying: pretty clearly the user would want the menu styling inherited by submenus, so this extra "option" is just an annoyance. I don't see an easy way to do it automatically, so just voicing this observation in case you think of a neat way.

Reply:
I'm having trouble coming up with a simpler solution